### PR TITLE
chore: use the correct syncOptions

### PR DESCRIPTION
**Pull request by Tuditi**
# Description of change

Update the types to use the correct sync options.

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes #issue_number`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Add a .change please

**Review by Thoralf-M — DISMISSED** [review]

File: .changes/fix-use-correct-sync-options.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+---
+"client-nodejs-binding": patch
```
Comment (Thoralf-M) [inline .changes/fix-use-correct-sync-options.md:3]: Wait, this should be wallet

Comment (Thoralf-M) [inline .changes/fix-use-correct-sync-options.md]:
```suggestion
"wallet-nodejs-binding": patch
```

**Review by Thoralf-M — APPROVED** [review]

File: .changes/fix-use-correct-sync-options.md (hunk 1 above)
Comment (Tuditi) [reply .changes/fix-use-correct-sync-options.md]: Whoops, didn't notice that. Thanks a lot!

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**